packaging/redhat_fedora/opensips-cli.spec

```diff
@@ -22,10 +22,16 @@ Requires: python36-sqlalchemy
 Requires: python36-mysql
 Requires: python36-pyOpenSSL
 %else
+%if 0%{?rhel} == 8
```

Comment:
`0%{?rhel} < 8` (and switch the `Requires`) would be better, in case CentOS 9+ keep the package names unchanged.

Reply:
Agree - fix is on the way...